packages/eslint-plugin-patternfly-react/lib/rules/import-tokens-icons.js

```diff
@@ -1,3 +1,23 @@
+/**
+ * @param specifier module specifier
+ * @param moduleName module name (@patternfly/react-tokens or @patternfly/react-icons)
+ * @returns string of non-treeshaken import
+ */
+function makeImport(specifier, moduleName) {
+  let res = `import ${specifier.local.name} from '`;
+  res += moduleName.replace(/\/dist\/(js|esm)/, '');
+  res += '/dist/js';
```

Comment:
only `/dist/js`?

Reply:
This function builds a path to the CJS module.

Ex: `import { some_token } from '@patternfly/react-tokens';` -> `import some_token from '@patternfly/react-tokens/dist/js/some_token';`